File: SEARCHING/LINEAR-SEARCH/C/LinearSearch.c
#include<stdio.h>
int main()
{
	int n;
	scanf("%d",&n);
	int arr[n];
	for(int i=0;i<n;i++)
	{
		scanf("%d",&arr[i]);
	}
	int find;
	int i;
	scanf("%d",&find);
	int flag = 0;
	for(i=0;i<n;i++)
	{
		if(arr[i]==find)
		{
			flag = 1;
			break;	
		}
	}
	if(flag ==0){
	printf("%d index of this",i);
	}
	else
	{
		printf("no element");
	}
		return 0;
}

File: searching/linear_search/linearSearch.py
def linear_search(arr, key):
    for j in range(len(arr)):
        if (arr[j] == key):
            return j
    return -1

 
my_list = [1, 3, 5, 7, 9]
k = 7

result = linear_search(my_list, k)

if(result == -1):
    print("Couldn't find the specified element")
else:
    print("Element found at index: ", result)
